build_tools/python_deploy/build_linux_packages.sh

```diff
@@ -186,26 +186,45 @@ function build_in_tree() {
   ccache -s
 }
 
+function _check_file_not_changed_by() {
+  # _check_file_not_changed_by <cmd> <file>
+  cmd="$1"
+  file="$2"
+  file_backup="/tmp/$(basename $file)"
+  file_new="/tmp/$(basename $file).new"
```

Comment:
Minor suggestion is to use the PWD which is more likely to be writable than /tmp/ which may or maynot have perms etc. We noticed similar failures with torch_mlir.compile() and we override writing to /tmp.

Reply:
Done.